client/src/components/DisplayTable.js

```diff
@@ -88,13 +92,17 @@ export class DisplayTable extends React.Component {
                             desc={descending && sort_by === tick}
                           />
                         </div>
-                        { !_.isUndefined(col_search[tick]) &&
+                        { _.includes(col_search_keys, tick) &&
                           <input
                             type="text"
                             id={tick}
                             style={{ width: "100%", fontWeight: "normal" }}
                             onChange={evt => {
-                              const new_query = _.clone(col_search);
+                              const new_query = _.chain(col_search_keys)
+                                .keyBy()
+                                .mapValues((key)=>col_search[key] || "")
```

Comment:
I do wish this was spaced a little bit better for readability `.mapValues( (key) => col_search[key] || "" )

Reply:
:+1: